src/kvstore/NebulaStore.cpp

```diff
@@ -955,7 +960,7 @@ ResultCode NebulaStore::setWriteBlocking(GraphSpaceID spaceId, bool sign) {
             }
             if (ret != ResultCode::SUCCEEDED) {
                  LOG(ERROR) << "Part sync failed. space : " << spaceId << " Part : " << part;
-                 return ret;
+                 continue;
```

Comment:
Why ignore sync errors?

Reply:
The error is ignored here because the meta call to the setWriteBlocking interface actually ignores the error.

Reply:
> I don't quite understand, if sync return error, it can't be ignored.

This is because in setWriteBlocking, it goes through all the parts and then sets the block flag, and finally sends a sync, where the sync must have a part of the failure (because the leader change), and I think this part of the failure can be ignored, because the failure of the machine will eventually call setWriteBlocking in the meta.

Reply:
I got it. I suppose we don't need to `sync` because this is a memory lock, and when meta `setWriteBlocking`, it send to all storage in a space. @bright-starry-sky , do you remember why we sync here?

Reply:
Yeah, I thought it was weird the first time I saw the code here, and when I tested it, sync must have returned an error because the current machine is definitely not the leader of all the parts.

Reply:
If so we could remove it, all failure should return to meta.